src/utility/validation.js

```diff
@@ -129,6 +129,34 @@ const SIGNIN_VALIDATION_RULES = [
     body('password').exists().isString().withMessage('password is required'),
 ];
 
+const CREATE_EVENT_VALIDATION_RULES = [
+    body('title').exists().isString().isLength({ max: 50 }).withMessage('title is required'),
+    body('content').exists().isString().isLength({ max: 1000 }).withMessage(' content is required'),
+    body('coverImage')
+        .custom((value, { req }) => {
+            if (!req.file) return false;
+            if (
+                req.file.mimetype.split('/')[0] !== 'image' ||
+                req.file.size > MAX_IMAGE_SIZE
+            ) {
+                return false;
+            }
+            return true;
+        })
+        .withMessage('coverImage must be an image less than 10MB'),
+    body('date').exists().isDate().withMessage('date is required'),
+    body('category').exists().isString()
```

Comment:
The category is changed to categories in the dev branch. It is an array now, make sure you update your validation accordingly. 
Something like: 
```
    body('categories')
        .exists()
        .isArray({ min: 1 })
        .withMessage('categories must be an unempty array')
        .custom((array) =>
            array.every(
                (category) =>
                    isString(category) &&
                    variables.CATEGORIES.includes(category)
            )
        )
        .withMessage('interests must be an array of valid interests'),
```